pkg/plugins/manager/manifest.go

```diff
@@ -207,6 +202,9 @@ func getPluginSignatureState(log log.Logger, plugin *plugins.PluginBase) (plugin
 				Status: plugins.PluginSignatureModified,
 			}, nil
 		}
+		if err := f.Close(); err != nil {
```

Comment:
If `if sum != hash {` above is true you probably want to close the file as well? If failing to copy the file, is the file automatically close otherwise might need to close the file there as well?

Reply:
Ah of course - will fixup! Thanks